packages/core/useSpeechSynthesis/index.ts

```diff
@@ -113,6 +113,11 @@ export function useSpeechSynthesis(text: MaybeComputedRef<string>, options: UseS
     utterance && synth!.speak(utterance.value)
   }
 
+  const isStop = () => {
```

Comment:
```suggestion
  const stop = () => {
```

It should be a verb... right?

Reply:
right..